src/core/csv/reader.cc

```diff
@@ -151,6 +154,17 @@ void GenericReader::init_nthreads(const py::Arg& arg) {
   }
 }
 
+
+void GenericReader::init_encoding(const py::Arg& arg) {
+  if (arg.is_none_or_undefined()) return;
+  encoding_ = arg.to_string();
+  if (!PyCodec_KnownEncoding(encoding_.c_str())) {
+    throw ValueError() << "Unknown encoding " << encoding_;
```

Comment:
Do we have documentation added somewhere for the new `encoding` parameter? I guess it would be worth to mention that it is case insensitive, since It seems that both `PyCodec_KnownEncoding()` and `PyCodec_IncrementalDecoder()` are case insensitive.

Reply:
added mention in the changelog